vehicle_interface/src/sd_vehicle_interface/sd_vehicle_interface.cpp

```diff
@@ -118,6 +118,8 @@ int main(int argc, char **argv)
 
 
     ros::Rate loop_rate(ROS_LOOP);
+	ros::Time autonomous_entry;
```

Comment:
It looks correct in my text editor but it's a tab instead of spaces so depending on the interpreter it could be displayed funny (like here)
```suggestion
    ros::Rate loop_rate(ROS_LOOP);
    ros::Time autonomous_entry;
```